androidHyperskillApp/src/main/java/org/hyperskill/app/android/home/view/ui/fragment/HomeFragment.kt

```diff
@@ -46,7 +46,7 @@ class HomeFragment :
         object : DefaultLifecycleObserver {
             override fun onResume(owner: LifecycleOwner) {
                 super.onResume(owner)
-                homeViewModel.onNewMessage(HomeFeature.Message.Init(forceUpdate = true))
+                homeViewModel.onNewMessage(HomeFeature.Message.Initialize(forceUpdate = true))
```

Comment:
Refactored from `Init` to `Initialize` because `init` is a reserved keyword in the Swift language and we cannot use it when generating Swift code for the shared module using moko-kswift.